lint/backend.py

```diff
@@ -13,7 +14,7 @@
 WILDCARD_SYNTAX = '*'
 
 
-def lint_view(view, hit_time, callback):
+def lint_view(view, hit_time, next):
```

Comment:
`next` is a built-in function.

Reply:
I know that. But it's also the canonical name for the continuation function.